src/main/scala/service/authorisation/AuthorisationService.scala

```diff
@@ -0,0 +1,63 @@
+package service.authorisation
```

Comment:
Work in progress, I've not plumbed this in yet. But the idea is that for any request, we'll look in the Cookie or the Header for a valid JWT.